src/aeso_ast_to_fcode.erl

```diff
@@ -783,6 +793,8 @@ op_builtins() ->
 
 builtin_to_fcode(require, [Cond, Msg]) ->
     make_if(Cond, {tuple, []}, {builtin, abort, [Msg]});
+builtin_to_fcode(chain_event, [Event]) ->
+    {def, event, [Event]};
```

Comment:
Any risk of name clash? I remember we talked about it 🤔

Reply:
`event` is a special name here, user defined functions are `{entrypoint, Name}` or `{local_fun, Name}`. It turns into `Chain.event` in the fate code.